sunpy/data/sample.py

```diff
@@ -11,7 +11,11 @@
 for _key in _sample_files:
     if os.path.isfile(os.path.join(_sampledata_dir, _sample_files[_key])):
         setattr(sys.modules[__name__], _key, os.path.join(_sampledata_dir, _sample_files[_key]))
+        print(os.path.join(_sampledata_dir, _sample_files[_key]))
```

Comment:
This print statement seems like diagnostic output rather than something a user should see; I think it should be removed.  Why would a user care what the filename is for a sample file or what its full path is?  After all, that's why we have the labels (e.g., `AIA_171_IMAGE`).